test/test_correctLik.jl

```diff
@@ -140,47 +142,39 @@ end
 @testset "snaq! in serial and in parallel w/ different probQR and propQuartets values" begin
   global tree = readnewick("((((6:0.1,4:1.5),9)1:0.1,8),10:0.1);")
   @test_throws ErrorException snaq!(tree, d) # some taxa are in quartets, not in tree
-  # originalstdout = stdout
-  # redirect_stdout(devnull)
   global net = readnewick("((((6:0.1,4:1.5)1:0.2,((7,60))11#H1)5:0.1,(11#H1,8)),10:0.1);")
   snaq!(net, d, # taxon "60" in net: not in quartets
   hmax=1, runs=1, Nfail=1, seed=1234, ftolRel=1e-2,ftolAbs=1e-2,xtolAbs=1e-2,xtolRel=1e-2)
   Random.seed!(42)
   global n1 = snaq!(currT, d, hmax=1, runs=2, Nfail=1, seed=123,
             ftolRel=1e-2,ftolAbs=1e-2,xtolAbs=1e-2,xtolRel=1e-2,
-            probQR = 0.75, propQuartets = 0.8, verbose=false)
+            probQR = 0.0, propQuartets = 0.4, verbose=false)
   addprocs(1)
   @everywhere using SNaQ
   Random.seed!(42)
   global n2 = snaq!(currT, d, hmax=1, runs=2, Nfail=1, seed=123,
              ftolRel=1e-2,ftolAbs=1e-2,xtolAbs=1e-2,xtolRel=1e-2,
              probQR = 0.75, propQuartets = 0.8)
-  # redirect_stdout(originalstdout)
   rmprocs(workers())
-  n3 = readsnaqnetwork("snaq.out")
+  Random.seed!(42)
+  global n3 = snaq!(currT, d, hmax=1, runs=2, Nfail=1, seed=123,
+             ftolRel=1e-2,ftolAbs=1e-2,xtolAbs=1e-2,xtolRel=1e-2,
+             probQR = 0.25, propQuartets = 0.6)
   
-  # propQuartets = 0.8 means that branch lengths may optimize to different values
-  for n in [n1, n2, n3]
-    for e in n.edge
-      e.length = -1.
-      e.gamma = -1.
-    end
-  end
   @test loglik(n3) > 0.0
-  # b/c of propQuartets the newicks will sometimes be different
-  @test hardwiredclusterdistance(n1, n2, false) == 0
-  @test hardwiredclusterdistance(n2, n3, false) == 0
+
+  # Networks are not necessarily the same, so can't do HWCD test
```

Comment:
could we test that we always get the same n1, the same n2, the same n3? that is, we hard code who n1 should be, and we test we get that network everytime we run line 149 (as a regular test for the output rather to each other)?